WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -284,11 +296,11 @@ public async Task RemoveInputAsync(InputsRemovalRequest request)
 			{
 				if (!Rounds.TryGetValue(request.RoundId, out var round))
 				{
-					throw new WabiSabiProtocolException(WabiSabiProtocolErrorCode.RoundNotFound);
```

Comment:
@yahiheb Please also this one:

`Logger.LogInfo($"{removedAliceCount} alices timed out and removed.");`

->

`Logger.LogInfo($"Round ({round.Id}): {removedAliceCount} alices timed out and removed.");`

Reply:
Changed to:
`round.LogInfo($"{removedAliceCount} alices timed out and removed.");`